app/src/components/notifications/NotificationBanner.vue

```diff
@@ -51,26 +52,28 @@ switch (props.notification.state) {
 
 <template>
     <div v-if="show" class="inset-x-0 top-0 z-50 text-zinc-900" :class="color">
-        <div class="flex items-center justify-between px-6 py-1">
-            <div class="flex items-center gap-2">
-                <component :is="icon" class="h-5 min-h-5 w-5 min-w-5" />
-                <div class="flex flex-col md:inline-block md:align-middle">
-                    <span class="text-md md:text-sm">{{ notification.title }}</span>
-                    <span v-if="notification.description" class="text-xs md:ml-3">{{
-                        notification.description
-                    }}</span>
+        <component :is="notification.routerLink ? RouterLink : 'div'" :to="notification.routerLink">
+            <div class="flex items-center justify-between px-6 py-1">
+                <div class="flex items-center gap-2">
+                    <component :is="icon" class="h-5 w-5 min-w-5" />
+                    <div class="flex flex-col md:inline-block md:align-middle">
+                        <span class="text-md md:text-sm">{{ notification.title }}</span>
+                        <span v-if="notification.description" class="text-xs md:ml-3">{{
+                            notification.description
+                        }}</span>
+                    </div>
                 </div>
-            </div>
 
-            <button
-                type="button"
-                @click="notification.id ? removeNotification(notification.id) : (show = false)"
-                class="h-6 min-h-6 w-6 min-w-6 cursor-pointer underline md:h-5 md:min-h-5 md:w-5 md:min-w-5"
-                data-test="banner-close-button"
-            >
-                <span class="sr-only">Close</span>
-                <XMarkIcon class="h-5 w-5" aria-hidden="true" />
-            </button>
-        </div>
+                <button
+                    type="button"
+                    @click="notification.id ? removeNotification(notification.id) : (show = false)"
+                    class="h-6 min-h-6 w-6 min-w-6 cursor-pointer underline md:h-5 md:min-h-5 md:w-5 md:min-w-5"
+                    data-test="banner-close-button"
+                >
+                    <span class="sr-only">Close</span>
+                    <XMarkIcon class="h-5 w-5" aria-hidden="true" />
+                </button>
+            </div>
+        </component>
```

Comment:
Because the button is inside the routerlink, clicking close will still bring me to the login page. The close button should probably be outside of the routerlink. Perhaps only the actual `description` should be clickable?

Reply:
I found also that. I can fix it